stacker_blueprints/empire/controller.py

```diff
@@ -0,0 +1,177 @@
+from troposphere import (
```

Comment:
Was this not `git mv`'d ?  Crazy that git didn't recognize that they are the same file, and this makes it really hard to review the changes.